src/journal/meson.build

```diff
@@ -0,0 +1,104 @@
+# -*- mode: meson -*-
+
+libsystemd_journal_internal_sources = files('''
+  audit-type.c
+  audit-type.h
+  catalog.c
+  catalog.h
+  compress.c
+  compress.h
+  journal-def.h
+  journal-file.c
+  journal-file.h
+  journal-send.c
+  journal-vacuum.c
+  journal-vacuum.h
+  journal-verify.c
+  journal-verify.h
+  lookup3.c
+  lookup3.h
+  mmap-cache.c
+  mmap-cache.h
+  sd-journal.c
+'''.split())
+
+############################################################
+
+audit_type_includes = [config_h,
+                       missing_h,
+                       'linux/audit.h']
+if libaudit.found()
```

Comment:
That's a left-over from before I stumbled upon the solution with `[]` and `if conf.get(..., 0) == 1`. But that is the "official" way, the wiki recommends using `[]` now (or it always did, and I didn't see it). Should be fixed now.